live-examples/js-examples/string/string-substr.html

```diff
@@ -0,0 +1,6 @@
+<pre>
+<code id="static-js">const str = 'Mozilla';
+console.log(str.substr(1,2)); // 'oz'
+console.log(str.substring(2,5)); // 'zilla'
+</code>
```

Comment:
It would be good to add a version that omits the second argument.

Reply:
It would also be nice to use whitespace here, like:

```
const str = 'Mozilla';

console.log(...
// expected output: ...


console.log(...
// expected output: ...
```

Reply:
Sorry, for some reason I thought I had to use the minimum number of lines.